### ISSUE TITLE
Cannot read property 'isSame' of null

### ISSUE THREAD

**Issue by dadwic** [mberneti/react-datepicker2#102]
Hi there,
This error occurs when the component is updated and receive props.

DatePicker.UNSAFE_componentWillReceiveProps
node_modules/react-datepicker2/dist/es/index.js:2927

 ```
  2924 | }, {
  2925 |   key: "UNSAFE_componentWillReceiveProps",
  2926 |   value: function UNSAFE_componentWillReceiveProps(nextProps) {
>2927 |     if ('value' in nextProps && (typeof nextProps.value === 'undefined' && typeof this.props.value !== 'undefined' || typeof nextProps.value !== 'undefined' && !nextProps.value.isSame(this.props.value))) {
       | ^  
  2928 |       this.setMomentValue(nextProps.value);
  2929 |     }
  2930 |
```

**Comment by mberneti** [comment]
@dadwic Despite I am working on refactoring and it fixes the issue, I'll check it soon.

**Comment by mberneti** [comment]
@dadwic Can you reproduce this error on the [codesandbox](http://codesandbox.io/) ?

**Comment by hoseinBL** [comment]
I have the same issue here. I don't want to have an initial value. I set `value` to `null` but I face this error:
Uncaught TypeError: Cannot read property 'isSame' of null

**Comment by mberneti** [comment]
@hoseinBL Thank you for reporting this issue, I'll work on it tonight.

**Comment by hoseinBL** [comment]
Any good news man?

**Comment by mberneti** [comment]
@hoseinBL Yes, this issue fixed on v3.3.4
